src/python/dxpy/scripts/dx.py

```diff
@@ -5636,6 +5855,94 @@ def register_parser(parser, subparsers_action=None, categories=('other', ), add_
+                            default=False, action='store_true')
+parser_archive.add_argument('-y', '--yes', dest='confirm', help='Do not ask for confirmation',action='store_false')
+parser_archive.add_argument('--no-recurse', dest='recurse',help=fill('When `path` refers to a single folder, this flag causes only files in the specified folder and not its subfolders to be archived.This flag has no impact when `path` input refers to a collection of files.', width_adjustment=-24), action='store_false')
+
+parser_archive.add_argument(
+    'path', 
+    help=fill('List of path(s) that should be archived. May refer to a single folder or specify one or more files in a single project.',width_adjustment=-24),
+    default=[], nargs='+').completer = DXPathCompleter() 
+parser_archive.set_defaults(func=archive)  
+register_parser(parser_archive, categories='fs')
+
+#####################################
+# unarchive
+#####################################
+
+parser_unarchive = subparsers.add_parser(
+    'unarchive', 
+    help='Requests for the specified set files or for the files in a single specified folder to be unarchived on the platform.',    
+    description=fill('Requests for the specified set files or for the files in a single specified folder to be unarchived on the platform.' +
+    'The requested copy will eventually be transitioned over to the live state while all other copies will move over to the archival state.') + 
+    '''
+
+EXAMPLES
+
+  ''' + 
+  fill('# unarchive 3 files in project "FirstProj" with project id project-B0VK6F6gpqG6z7JGkbqQ000Q', width_adjustment=-2, subsequent_indent='  ') + 
+  '''
+  $ dx archive  FirstProj:file-B0XBQFygpqGK8ZPjbk0Q000Q FirstProj:/path/to/file1 project-B0VK6F6gpqG6z7JGkbqQ000Q:/file2
+
+  ''' + 
+  fill('# unarchive all files recursively in project-B0VK6F6gpqG6z7JGkbqQ000Q', width_adjustment=-2, subsequent_indent='  ') + '''
+  $ dx unarchive project-B0VK6F6gpqG6z7JGkbqQ000Q:/
+  ''',
+                                               formatter_class=argparse.RawTextHelpFormatter,
+                                               parents=[all_arg],
+                                               prog='dx unarchive'
+                                               )
+
+parser_unarchive.add_argument('--rate', help=fill('The speed at which all files in this request are unarchived.', width_adjustment=-24) + '\n'+ fill('- Azure regions: {Expedited, Standard}', width_adjustment=-24,initial_indent='  ') + '\n'+ 
+fill('- AWS regions: {Expedited, Standard, Bulk}', width_adjustment=-24,initial_indent='  '), choices=["Expedited", "Standard", "Bulk"], default="Standard")
+
+parser_unarchive_output = parser_unarchive.add_argument_group(title='Output').add_mutually_exclusive_group()
+parser_unarchive_output.add_argument('-q', '--quiet', help='Do not print extra info messages', action='store_true')
+parser_unarchive_output.add_argument(
```

Comment:
should contain
```
If -q option is specified, prints nothing. Otherwise,   if --dry-run is not specified, prints "Tagged <> files for unarchival, totalling <> GB, costing <format the value returned by api call using project's currency setting>" and
    if --dry-run is  specified, prints "Would tag <> files for unarchival, totalling <> GB, costing <>"
```